addons/external-player-button/userscript.js

```diff
@@ -0,0 +1,55 @@
+﻿export default async function ({ addon, global, console }) {
+  addon.settings.addEventListener("change", () => console.log("changed!"));
```

Comment:
Is this necessary?

Reply:
I dunno, I just copied another extension's stuff and kept it for ... um ... reasons.

Reply:
XD

Reply:
For quite obvious reasons which are ... umm ... 
Next question!